config/lighthouse.php

```diff
@@ -0,0 +1,278 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | GraphQL Endpoint
+    |--------------------------------------------------------------------------
+    |
+    | Set the endpoint to which the GraphQL server responds.
+    | The default route endpoint is "yourdomain.com/graphql".
+    |
+    */
+
+    'route_name' => 'graphql',
```

Comment:
Should we set this to something custom from the off? I think it would be nice to have `/api/` or similar. Whatever we decide, let's get into the habit of using it early?

Reply:
I think API is sensible :)